s2member-pro/includes/classes/sc-last-payment-in.inc.php

```diff
@@ -63,7 +63,7 @@ public static function sc_last_payment_details($attr = array(), $content = '', $
             // Initialize Last Payment details/output date format.
 
             $time = null; // Initialize the time calculation.
-            if($last_payment['time']) // // Do we have a time to work with?
+            if($last_payment_time) // // Do we have a time to work with?
                 {
                     $time = new DateTime(date('Y-m-d H:i:s', $last_payment['time']));
```

Comment:
@renzms Look for other instances of `$last_payment['time']` in this file and change them to `$last_payment_time`.

See also: Example # 6 in this article: http://php.net/manual/en/language.types.array.php

What is there now is trying to find the `time` index in an associative array named `$last_payment`, when that's actually _not_ was has been defined at the stop of the shortcode handler. Instead of an array, the variable is simply defined as `$last_payment_time`.